frontend/src/Components/Explore.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from "react";
+import axios from "axios";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TableSortLabel from "@mui/material/TableSortLabel";
+import TextField from "@mui/material/TextField";
+import "./Explore.css";
+
+const Explore = () => {
+  const [activeFunds, setActiveFunds] = useState([]);
+  const [orderBy, setOrderBy] = useState("");
+  const [order, setOrder] = useState("asc");
+  const [searchText, setSearchText] = useState("");
+
+  const handleClick = async (e) => {
+    await axios.get("http://localhost:8000/apis/v1/funds").then((resp) => {
```

Comment:
Add `http://localhost:8000` to `.env` and use `process.env` here, later when you deploy backend you only have to do changes in one file.

Reply:
Okay, I will apply the changes and merges.